### PR TITLE
First draft of Chapter 1

### PR DESCRIPTION
**Pull request by jhurdlow**
Completed initial conversion of Chapter 1: OCaml Latex to F# Markdown. Needs MacOS and Linux install instructions, curly-quote replacement (or removal), etc... Apologies for not getting these changes flattened, got tired of fighting with Git.

### CODE REVIEW

File: 01-the_way_of_the_program/01.2.1_windows.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Visual Studio Code (VSCode) has a very nice integration with FSI. Typing F# code in VSCode, selecting it, then hitting Alt-Enter will send that line to FSI.
+
+Once in FSI, you can open a script by using the `#load "filename";;` directive:
+
+```
+> #load "myscript.fsx";;
+[Loading C:\Users\me\myscript.fsx]
+```
+
+Note that FSI's **current directory** (the directory in which F# will look to find files to run, outside of the standard F# directories) will be the current directory when you ran it within the terminal. You can change your current directory using the `#cd dirname;;` directive.
```
Comment (jeroldhaas) [inline 01-the_way_of_the_program/01.2.1_windows.md:15]: May want to mention `__SOURCE_DIRECTORY__` here. However, perhaps it's too soon?

Comment (cloudRoutine) [reply 01-the_way_of_the_program/01.2.1_windows.md:15]: that can wait for a more in depth exploration into scripting

**Comment by cloudRoutine** [comment]
@jhurdlow Fantastic! I've wanted to get back to this, but working on F# tooling seems to monopolize all of my free time.  

I'm working on improving [F# Formatting](https://github.com/tpetricek/FSharp.Formatting) so that it can produce nice CSS tooltips for code snippets. I really want to get those into this book.

Thanks a bunch for this 👍

**Comment by jeroldhaas** [comment]
@cloudRoutine this is a bit more "meta-", but, are we cool with submitting PRs directly to `master`, or should we instead formalize submissions to feature-branches, as in a more gitflow-style workflow?

**Comment by cloudRoutine** [comment]
right onto master is fine

**Comment by jhurdlow** [comment]
Yep, glad to do what I can. I was going to start doing it on my own, then ran across this project, so I figured It'd be better to pitch in here. I'm working on Chapter 2 now... first draft should be done in a couple days I expect.

**Pull request merged**